modules/tinymce/package.json

```diff
@@ -15,16 +15,27 @@
   "engines": {
     "node": ">=0.10.26"
   },
-  "main": "tinymce.js",
+  "main": "js/tinymce/tinymce.js",
+  "types": "js/tinymce/tinymce.d.ts",
   "scripts": {
+    "build": "grunt",
     "test": "grunt test",
     "lint": "grunt eslint",
     "silver-test": "grunt bedrock-auto:silver",
     "silver-test-manual": "grunt bedrock-manual:silver"
   },
   "dependencies": {
-    "dompurify": "^3.0.5",
+    "dompurify": "3.0.5",
     "prismjs": "^1.27.0",
     "tslib": "^2.0.0"
+  },
+  "devDependencies": {
+    "@ephox/acid": "^6.0.0",
+    "@ephox/agar": "^8.0.0",
```

Comment:
Shouldn't this also need a dependency on mcagar?

Reply:
No, it can't - mcagar depends on tinymce. It's a complicated mess previously hidden by `tsc -b`.